pkg/agent/controller/policy/policy_controller.go

```diff
@@ -306,6 +306,11 @@ func (r *Reconciler) completePolicy(policy *securityv1alpha1.SecurityPolicy) ([]
 		return nil, err
 	}
 
+	// if apply to is nil or empty, add all ips
+	if len(policy.Spec.AppliedTo) == 0 {
```

Comment:
Do we need to check nil before get length?

Reply:
len(nil) will return 0

Reply:
OK